LQResultProducer.py

```diff
@@ -4515,20 +4514,24 @@ def SysTable(optimlog, selection_uujj,selection_uvjj,NormalDirectory, weight,sys
 	rglobalb = 1.0
 
 
-	if 'ZNORMup' in sysmethod :   
-		#rz *= 1.1# adding this to cover st/muj kinematic difference
+	if 'ZNORM' in sysmethod and 'up' in sysmethod :   
+		rz *= 1.1# adding this to cover st/muj kinematic difference
 		rz += _e_rz 
-	if 'ZNORMdown' in sysmethod : 
-		#rz *= 0.9# adding this to cover st/muj kinematic difference
+	if 'ZNORM' in sysmethod and 'down' in sysmethod : 
+		rz *= 0.9# adding this to cover st/muj kinematic difference
 		rz += -_e_rz 
-	if 'WNORMup' in sysmethod :     
+	if 'WNORM' in sysmethod and 'up' in sysmethod :     
 		rw += _e_rw
-	if 'WNORMdown' in sysmethod :  
+	if 'WNORM' in sysmethod and 'down' in sysmethod :  
 		rw += -_e_rw 
-	if 'TTNORMup' in sysmethod :  
+	if 'TTNORM' in sysmethod and 'up' in sysmethod :  
 		rt += _e_rt
-	if 'TTNORMdown' in sysmethod :  
+	if 'TTNORM' in sysmethod and 'down' in sysmethod :  
 		rt += -_e_rt 	
+	if 'VVNORM' in sysmethod and 'up' in sysmethod :
+		rvv = Rvv_uujj_up
```

Comment:
so im inclined to think we should just treat it the same as the DY and TT, that is to say, just vary up and down by the statistical error on the scale factor. Then we can check the stability in the same way as DY/TT and apply a systematic if needed (hopefully not)